include/OpenSpaceToolkit/Astrodynamics/Conjunction/Messages/CCSDS/CDM.hpp

```diff
@@ -13,7 +13,7 @@
 #include <OpenSpaceToolkit/Core/Types/Shared.hpp>
 #include <OpenSpaceToolkit/Core/Types/String.hpp>
 
-#include <OpenSpaceToolkit/Mathematics/Objects/Matrix.hpp>
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
```

Comment:
Is that an automated change?

Reply:
nope I changed it manually cause there's actually nothing in the `Matrix.hpp` file, its all in the `Vector.hpp` file